examples/flutter_gallery/lib/demo/pesto_demo.dart

```diff
@@ -569,266 +572,146 @@ class RecipeStep {
-      const RecipeIngredient(amount: '1 tsp', description: 'Lemon juice'),
-      const RecipeIngredient(amount: '1/2 cup', description: 'Sun dried tomatoes'),
     ],
     steps: const<RecipeStep>[
-      const RecipeStep(duration: '3 min', description: 'Stir'),
-      const RecipeStep(duration: '45 min', description: 'Cook'),
+      const RecipeStep(duration: '15 min', description: 'Blend'),
     ],
   ),
   const Recipe(
-    name: 'Oven Sausage',
+    name: 'Cherry Pie',
     author: 'Sandra Adams',
-    ingredientsImagePath: 'food/icons/meat.png',
+    ingredientsImagePath: 'pesto/main.png',
     ingredientsImagePackage: _kGalleryAssetsPackage,
-    description: 'Robust cuts of portuguese sausage add layers of flavour. Bake or fry until sausages are slightly browned and with a crispy skin. Serve warm and with cuts of pineapple for a delightful mix of sweet and savory flavour. This is the perfect dish after a swim in the sea.',
-    imagePath: 'food/image4.jpg',
-    imagePackage: _kGalleryAssetsPackage,
+    description: 'Sometimes when you\'re lacking cheer in your life you can jumpstart your day with some cherry pie. Dessert for breakfast is perfectly viable. We won\'t judge.',
+    imagePath: 'assets/image4.png',
+//    imagePackage: _kGalleryAssetsPackage,
     ingredients: const<RecipeIngredient>[
-      const RecipeIngredient(amount: '1 1/2 lbs', description: 'Linguisa'),
-      const RecipeIngredient(amount: '1 lbs', description: 'Pineapple or other fresh citrus fruit'),
+      const RecipeIngredient(amount: '1', description: 'Pie crust'),
+      const RecipeIngredient(amount: '4 cups', description: 'Fresh or frozen cherries'),
+      const RecipeIngredient(amount: '1 cup', description: 'Granulated sugar'),
+      const RecipeIngredient(amount: '4 tbsp', description: 'Cornstarch'),
+      const RecipeIngredient(amount: '1½ tbsp', description: 'Butter'),
     ],
     steps: const<RecipeStep>[
-      const RecipeStep(duration: '3 min', description: 'Stir'),
-      const RecipeStep(duration: '45 min', description: 'Cook'),
+      const RecipeStep(duration: '15 min', description: 'Mix'),
+      const RecipeStep(duration: '1hr 30 min', description: 'Bake'),
     ],
   ),
   const Recipe(
-    name: 'Chicken tostadas',
+    name: 'Spinach Salad',
     author: 'Peter Carlsson',
-    ingredientsImagePath: 'food/icons/spicy.png',
+    ingredientsImagePath: 'pesto/spicy.png',
     ingredientsImagePackage: _kGalleryAssetsPackage,
-    description: 'Crisp flavours and a bit of spice make this roasted chicken dish an easy go to when cooking for large groups. Top with Baja sauce for an extra kick of spice.',
-    imagePath: 'food/image5.jpg',
-    imagePackage: _kGalleryAssetsPackage,
+    description: 'Your favorite leafy green is back. Paired with fresh sliced onion, it\'s ready to tackle any dish, whether it be a salad or an egg scramble.',
+    imagePath: 'assets/image5.png',
+//    imagePackage: _kGalleryAssetsPackage,
```

Comment:
Not sure why you'd want the imagePackage attributes to be left out, but if that's intentional, then delete the line (here and elsewhere)